src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -57,6 +58,8 @@ internal static SourcePropertySymbol Create(SourceMemberContainerTypeSymbol cont
                 diagnostics,
                 out _);
 
+            // PROTOTYPE(partial-properties): it's unclear if the subtle difference between former 'isAutoProperty' and '!accessorsHaveImplementation' matters
```

Comment:
It makes me nervous to punt on this question as it'll be harder to analyze any change here in a future PR (harder to compare against original code). Can we sort this out now?

As far as I can tell, the two are opposite, except for the case of a property without accessor. 
There's only a couple of places where the flag gets used, so a couple of tests should be able to nail this down.